scripts/check-consistency.sh

```
@@ -375,6 +413,15 @@ grep -Fq "${ROLLOUT_PHASE_WORK_PERCENT}%" docs/index.html &&
 grep -Fq "${ROLLOUT_PHASE_6_PERCENT}%" docs/index.html &&
   echo "  OK Rounded Phase 6 percentage" ||
   { echo "  FAIL Rounded Phase 6 percentage"; ERRORS=$((ERRORS+1)); }
+grep -Fq "${ROLLOUT_PHASE_7_PERCENT}%" docs/index.html &&
+  echo "  OK Rounded Phase 7 percentage" ||
+  { echo "  FAIL Rounded Phase 7 percentage"; ERRORS=$((ERRORS+1)); }
+grep -Fq "Phase 7 is ${ROLLOUT_PHASE_7_COMPLETED}/${ROLLOUT_PHASE_7_TOTAL}" README.md &&
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)